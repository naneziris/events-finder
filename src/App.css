.App {
  text-align: center;
}
.App-logo {
  height: 150px;
  pointer-events: none;
}
.header {
	display: flex;
    justify-content: center;
    align-items: baseline;
    border-bottom: 10px solid
}
.header__title {
	font-size: 2rem;
    background: rgb(39, 73, 109);
    margin-left: 2rem;
    position: relative;
    z-index: 2;
    transform: skew(-7deg);
	text-transform: uppercase;
    padding: 0.5rem 1rem;
}
.header__subtitle {
	margin-left: 30px;
}

/* === GENERAL === */
header {
	margin: 50px 0;
}
body {
	font-family: Helvetica, sans-serif;
	background-color: rgb(0, 144, 158);
	min-height: 100vh;
	font-size: calc(10px + 2vmin);
	color: rgb(218, 225, 231);
}
main {
  width: 90%;
  max-width: 1200px;
  margin: 0 auto;
  padding-bottom: 50px;
}

/* === GLOBAL === */

/* Remove padding and margin */ 
*  
    { 
    margin: 0; 
    padding: 0; 
    border: 0; 
}
li {
  list-style-type: none;
}
h1,h2,h3,h4,h5,h6 {
	-webkit-font-smoothing: antialiased;
	text-rendering: optimizelegibility;
	font-weight: bold;
}
h1 {
	font-size: 22px;
}
h2 {
  font-size: 18px;
}
h3 {
  font-size: 17px;
}
h4 {
	font-size: 16px;
}
h5 {
	font-size: 15px;
}
p {
	font-size: 14px;
    font-weight: 300;
    padding: 10px 0;
    line-height: 20px;
}
p.lead {
	font-weight: normal;
}
a {
	color: rgb(218, 225, 231);
	text-decoration: none;
}
a:hover {
	text-decoration: none;
}
button, input, .btn, a.btn {
	-webkit-font-smoothing: antialiased;
	text-rendering: optimizelegibility;
}
.btn {
	font-weight: bold;
}
button.btn:focus {
	outline: none !important;
}
img {
	max-width: 100%;
}
