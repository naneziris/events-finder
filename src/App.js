import React from 'react';
import './App.css';
import EventsList from './components/EventsList';

function App() {
  return (
    <div className="App">
      <header className="App-header">
      <div className="header">
        <h1 className="header__title">
          <a href="/">Events Finder</a>
        </h1>
        <h2 className="header__subtitle">Find upcoming events in USA</h2>
      </div>
      </header>
      <main>
        <EventsList />
      </main>
    </div>
  );
}

export default App;
