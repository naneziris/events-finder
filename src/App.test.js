import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

it('Should match the snapshot', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
})
