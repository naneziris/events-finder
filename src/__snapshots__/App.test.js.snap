// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match the snapshot 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <div
      className="header"
    >
      <h1
        className="header__title"
      >
        <a
          href="/"
        >
          Events Finder
        </a>
      </h1>
      <h2
        className="header__subtitle"
      >
        Find upcoming events in USA
      </h2>
    </div>
  </header>
  <main />
</div>
`;
